test/s3-benchmarks/src/main/java/software/amazon/awssdk/s3benchmarks/BaseTransferManagerBenchmark.java

```diff
@@ -28,46 +33,39 @@
 import software.amazon.awssdk.core.async.AsyncResponseTransformer;
 import software.amazon.awssdk.services.s3.S3Client;
 import software.amazon.awssdk.services.s3.model.GetObjectRequest;
-import software.amazon.awssdk.services.s3.model.HeadObjectResponse;
 import software.amazon.awssdk.services.s3.model.PutObjectRequest;
 import software.amazon.awssdk.testutils.RandomTempFile;
-import software.amazon.awssdk.transfer.s3.S3TransferManager;
 import software.amazon.awssdk.transfer.s3.internal.S3CrtAsyncClient;
 import software.amazon.awssdk.utils.Logger;
 
 public abstract class BaseTransferManagerBenchmark implements TransferManagerBenchmark {
-    protected static final int WARMUP_ITERATIONS = 10;
-    protected static final int BENCHMARK_ITERATIONS = 10;
 
     private static final Logger logger = Logger.loggerFor("TransferManagerBenchmark");
     private static final String WARMUP_KEY = "warmupobject";
 
-    protected final S3TransferManager transferManager;
     protected final S3CrtAsyncClient s3;
     protected final S3Client s3Sync;
     protected final String bucket;
     protected final String key;
     protected final String path;
-    private final File file;
+    protected final int warmupConcurrency;
+    protected final File tmpFile;
 
     BaseTransferManagerBenchmark(TransferManagerBenchmarkConfig config) {
         logger.info(() -> "Benchmark config: " + config);
-        Long partSizeInMb = config.partSizeInMb() == null ? null : config.partSizeInMb() * 1024 * 1024L;
+        Long partSizeInMb = config.partSizeInMb() == null ? null : config.partSizeInMb() * MB;
         s3 = S3CrtAsyncClient.builder()
                              .targetThroughputInGbps(config.targetThroughput())
                              .minimumPartSizeInBytes(partSizeInMb)
                              .build();
         s3Sync = S3Client.builder()
                          .build();
-        transferManager = S3TransferManager.builder()
-                                           .s3ClientConfiguration(b -> b.targetThroughputInGbps(config.targetThroughput())
-                                           .minimumPartSizeInBytes(partSizeInMb))
-                                           .build();
```

Comment:
I removed S3TransferManager and used the internal S3CrtClient for testing because S3TransferManager doesn't allow customizing S3 client , and we can't reuse the same S3 crt client.